### ISSUE TITLE
Ingest all the uncatalogued accessions through Archivematica

### ISSUE THREAD

**Issue by alexwlchan** [wellcomecollection/platform#4092]
Previously fixed things:

- Fix downloads in the "Archival storage" tab
- Kick ClamAV off the shared instance
- Look at using bulk Elasticsearch index APIs

**Comment by alexwlchan** [comment]
Waiting for a decision from the archivists about whether we want to use disk images for everything or just the big packages.

**Comment by alexwlchan** [comment]
Sticking some progress on this ticket:

* We're going to use disk images for really big accessions (>5000 files), but for other things Archivematica is coping okay, if quite slow.

* We've had to make a bunch of changes to Archivematica along the way, including:

    - tweaking timeouts
    - updating/reverting stuff that was changed when we did the rebase; in particular a bunch of async code got removed that we were relying on
    - tweaking CPU/memory limits to make it more robust

  I still don't fully understand all the issues we've had, but it's good enough for our use.

* I have a spreadsheet of all the 37 existing accessions we need to migrate (at least, when we started this project). It gives you an idea of where things currently stand:
    
![Screenshot 2020-03-03 at 12 27 56](https://user-images.githubusercontent.com/301220/75775786-9eb75700-5d4a-11ea-8874-029d7f51bf43.png)

Archivematica is fairly stable at this point, so I'm going to close this ticket and create individual tickets for the remaining accessions, which are moving more slowly.
